test/distributed/test_functional_api.py

```diff
@@ -0,0 +1,249 @@
+# Owner(s): ["oncall: distributed"]
+
+import sys
+import torch
+import torch.distributed as dist
+import torch.distributed._functional_collectives as ft_c
+import torch.distributed.distributed_c10d as c10d
+import torch.distributed._tensor as dt
+
+if not dist.is_available():
+    print("Distributed not available, skipping tests", file=sys.stderr)
+    sys.exit(0)
```

Comment:
is it more idiomatic to mark the test cases with some decorator that requires a feature or skips the test?  maybe there isn't a good one though?